src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -947,4 +947,50 @@ public void deleteNonExistentNodeTest() throws NoResponseException, XMPPErrorExc
             assertEquals(StanzaError.Condition.item_not_found, e.getStanzaError().getCondition());
         }
     }
+
+    /**
+     * Assert that the server send a notification to subscribers when deleting a
+     * node that exist
+     * 
+     * <p>
+     * From XEP-0060 § 8.4.1:
+     * </p>
+     * <blockquote> In order to delete a node, a node owner MUST send a node
```

Comment:
This isn't the correct quote. It does not describe what you're testing here!

Reply:
This is not yet the complete test. It's just a step

Reply:
I know. Just leave this comment open until you fixed it. :)

Reply:
The quote still is not correct.

Reply:
I have already changed the quote